Author: Thomas Witt
Upstream-status: Pending
Reason: This fixes a linker error where multiple definitions of
        light_loglevel collided.


diff -ru a/light-1.2/src/helpers.c b/light-1.2/src/helpers.c
--- a/light-1.2/src/helpers.c	2018-09-23 20:37:07.000000000 +0200
+++ b/light-1.2/src/helpers.c	2021-06-07 16:39:05.490068470 +0200
@@ -9,6 +9,7 @@
 #include <errno.h> // errno
 #include <libgen.h> // dirname 
 
+light_loglevel_t light_loglevel;
 
 bool light_file_read_uint64(char const *filename, uint64_t *val)
 {
diff -ru a/light-1.2/src/helpers.h b/light-1.2/src/helpers.h
--- a/light-1.2/src/helpers.h	2018-09-06 00:24:09.000000000 +0200
+++ b/light-1.2/src/helpers.h	2021-06-07 16:38:51.487199835 +0200
@@ -21,7 +21,7 @@
     LIGHT_NOTE_LEVEL
 } light_loglevel_t;
 
-light_loglevel_t light_loglevel;
+extern light_loglevel_t light_loglevel;
 
 #define LIGHT_LOG(lvl, fp, fmt, args...)\
     if (light_loglevel >= lvl)\

